packages/engine/src/store/types.ts

```diff
@@ -80,9 +80,12 @@ export type Nodes = { [key: string]: Node }
 
 export type NodeValue = number | boolean | string
 export type NodeValues = { [key: string]: NodeValue }
+export type NodeValueType = Node['valueType']
```

Comment:
We still have `NodeValueType`, which evaluates to a union of all node value types 

```ts
 "string" | "number" | "boolean" | "enum" | "vector3" | "rgb"
```